### ISSUE TITLE
Empty line in line protocol body causes rest of file to be ignored

### ISSUE THREAD

**Issue by mark-rushakoff** [influxdata/influxdb#7240]
### Bug report

__System info:__ 

OSX, built from a0004b68577cdbfc921463404eab839c50f14a0c

__Steps to reproduce:__

Run this shell script:

```sh
#!/bin/bash

curl -s -XPOST http://localhost:8086/query --data-urlencode "q=CREATE DATABASE test"

POINTS_FILE=$(mktemp)

echo "foo,s=A n=0i" >> "$POINTS_FILE"
echo "foo,s=B n=1i" >> "$POINTS_FILE"

curl -XPOST "http://localhost:8086/write?db=test" --data-binary "@$POINTS_FILE"

echo "This should have count = 2:"
curl "http://localhost:8086/query?db=test" --data-urlencode "q=SELECT count(n) FROM foo"

echo > "$POINTS_FILE" # By putting a newline at the start of the file, the rest of the file will not be parsed.
echo "foo,s=A n=2i" >> "$POINTS_FILE"
echo "foo,s=B n=3i" >> "$POINTS_FILE"

curl -XPOST "http://localhost:8086/write?db=test" --data-binary "@$POINTS_FILE"

echo "This should have count = 4:"
curl "http://localhost:8086/query?db=test" --data-urlencode "q=SELECT count(n) FROM foo"
```

__Expected behavior:__ 

Count is 2 on the first query and 4 on the second query.

__Actual behavior:__ 

Count is 2 on both queries. The second write is cleanly accepted, with no indication to the user that there was any kind of parse error. It would be nice if the response, or at least a header, indicated how many points were parsed, so that the developer could more easily diagnose that not all points were accepted properly.

Offending line is here: https://github.com/influxdata/influxdb/blob/a0004b68577cdbfc921463404eab839c50f14a0c/models/points.go#L169-L171

**Comment by jsternberg** [comment]
I think that should be a continue. @jwilder can we get this into RC2 or should it go into the final release for 1.0?

**Comment by jsternberg** [comment]
Fixed via #7241.
